src/components/dashboard-shortcut/dashboard-shortcut.tsx

```diff
@@ -0,0 +1,110 @@
+import { Component, Prop, h } from '@stencil/core';
+
+/**
+ * This component can be used on places such as a start page or a dashboard.
+ * Clicking on the component should navigate the user to a new screen,
+ * to which you need to provide a URL, using the `href` property.
+ *
+ * By default, this navigation will happen within the same browser tab.
+ * However, it is possible to override that behavior, using the `target` property.
+ *
+ * @exampleComponent limel-example-dashboard-shortcut
+ * @exampleComponent limel-example-dashboard-shortcut-notification
+ * @exampleComponent limel-example-dashboard-shortcut-styling
+ */
+
+@Component({
+    tag: 'limel-dashboard-shortcut',
```

Comment:
We decided to call this "shortcut" not "dashboard shortcut"!

If we wanna keep the word "dashboard", the other component will also need to be called "dashboard info tile".

Dashboard is where this component is supposed to be used, not what the component is. Right?

Reply:
I agree with dropping the `dashboard`. But I'm not so sure I like `limel-shortcut`. That seems a bit too generic to describe this component. How about something like `limel-app-shortcut`, `limel-app-button`…?

Reply:
Well, my choice of name comes from this very old memory 😄 👇 
![image](https://user-images.githubusercontent.com/35954987/199513329-64e9bb31-7c77-4d9c-acff-674b2d98a85d.png)

app-shortcut 🤔 this could be really shortcut to anything, app, website, "ugly modal", a pane/view/bookmark in the app. I don't know. I feel shortcut is OK :)

Reply:
Yeah, I feel like `limel-shortcut` has grown on me 👍